README.md

```diff
@@ -56,6 +56,27 @@ This package also provides its own set of CSS styles, based on [those used by GO
 
 The class names used also differ, each prefixed with `govspeak-`. Therefore a `govspeak` or `gem-c-govspeak` class is not needed on any parent container.
 
+However, if you wish to generate class names that match those from the Govspeak
+ruby gem then you can pass the `govspeakGemCompatibility` option to marked.  For
+example:
+
+```
+marked.setOptions({ govspeakGemCompatibility: true })
+
+marked('%This is a warning callout%')
+```
+
+will output:
+
+```html
+<div class="application-notice help-notice" role="note" aria-label="Warning">
+  <p>This is a warning callout</p>
+</div>
+```
+
+**Note: If you use this option, you will need to provide the styles.  This package does
+not contain the CSS styles for these classes**
+
```

Comment:
```suggestion
### Using original Govspeak gem classes

If you wish to generate class names that match those from the Govspeak Ruby gem, you can pass the `govspeakGemCompatibility` option to marked. For
example:

```js
marked.setOptions({ govspeakGemCompatibility: true })

marked('%This is a warning callout%')
```

will output:

```html
<div class="application-notice help-notice" role="note" aria-label="Warning">
  <p>This is a warning callout</p>
</div>
```

> [!NOTE]\
> You will need to provide your own styles if you use this option.
```

Reply:
Hmmm, not sure if this suggested change is coming through correctly?

Reply:
Yeah, I was just about to comment to say that

Reply:
Is it because the suggested change is wrapped in "```" and so the first "```" after the js block is closing the suggestion?

Reply:
Okay if I merge this PR and make the change myself?

Reply:
Sure.  I was just making the change myself locally, but doing it your way saves dismissing the approval.